config/webpack/development.js

```diff
@@ -16,4 +16,17 @@ environment.plugins.append(
   })
 )
 
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin')
+const isWebpackDevServer = process.env.WEBPACK_DEV_SERVER
+if (isWebpackDevServer) {
+  environment.plugins.append(
+    'ReactRefreshWebpackPlugin',
+    new ReactRefreshWebpackPlugin({
+      overlay: {
+        sockPort: 3035,
```

Comment:
This port is configurable via the development-environment (because not having that broke @SleeplessByte's world), so that config needs to get into here somehow.

Reply:
Yes

Reply:
Currently, the port is also hard coded inside `config/webpacker.yml`. I think this is the file that should be configurable via the development-environment. And then this file would then get the port from `config/webpacker.yml`.